projects/igniteui-angular/src/lib/directives/for-of/for_of.directive.ts

```diff
@@ -168,6 +168,14 @@ export class IgxForOfDirective<T, U extends T[] = T[]> extends IgxForOfToken<T,U
     @Input()
     public igxForContainerSize: any;
 
+    /**
+     * @hidden
+     * @internal
+     * Initial chunk size if no container size is passed. If container size is passed then the igxForOf calculates its chunk size
+     */
+    @Input()
+    public igxForInitialChunkSize: any;
```

Comment:
I guess existing combo tests somewhat cover the use of `igxForInitialChunkSize` but it wouldn't hurt to have at least one unit test covering this just for the for-of